docs/rules/no-async-in-computed-properties.md

```diff
@@ -112,7 +112,9 @@ Nothing.
 
 ## :books: Further Reading
 
-- [vue-async-computed](https://github.com/foxbenjaminfox/vue-async-computed)
+- [vue-async-computed]
+
+[vue-async-computed]: https://github.com/foxbenjaminfox/vue-async-computed
```

Comment:
Again, why extract the link target here?

Reply:
Since I introduced `@eslint/mardkown`, the following code has been flagged by `markdown/no-missing-label-refs`.

https://github.com/vuejs/eslint-plugin-vue/blob/a198b52b0ac26c8b8069fef1e295758ef7e5d932/docs/rules/no-async-in-computed-properties.md?plain=1#L16

I changed it to use the link definition to resolve the warning.